test/observable/CollectionSpec.js

```diff
@@ -112,6 +112,17 @@ define([
 
                 expect(collection.sizeOf()).toEqual(2);
                 expect(collection._collection).toEqual(['a value','c value']);
+
+                collection.remove('never gonna give you up');
+                expect(collection.sizeOf()).toEqual(2);
+                expect(collection._collection).toEqual(['a value','c value']);
```

Comment:
try to remove something that doesn't exist in the list, things should remain unchanged